packages/@lwc/integration-karma/test/custom-elements/index.spec.js

```diff
@@ -79,15 +88,17 @@ if (SUPPORTS_CUSTOM_ELEMENTS) {
     });
 
     describe('patched registry', () => {
-        it('throws error for unsupported "extends" option', () => {
-            expect(() => {
-                customElements.define('x-unsupported-extends', class extends HTMLElement {}, {
-                    extends: 'button',
-                });
-            }).toThrowError(
-                'NotSupportedError: "extends" key in customElements.define() options is not supported.'
-            );
-        });
+        if (window.lwcRuntimeFlags.ENABLE_SCOPED_CUSTOM_ELEMENT_REGISTRY) {
```

Comment:
This is a pattern I started with `ENABLE_NATIVE_CUSTOM_ELEMENT_LIFECYCLE`. I feel it's best to have one source of truth – the `lwcRuntimeFlags`.

Otherwise `process.env.FOO` and `lwcRuntimeFlags.FOO` could get out of sync with each other.